makefile

```diff
@@ -259,7 +262,11 @@ ${OBJS_EXE} ${OBJS_DLL} ${OBJS_ALL}: ${OBJPATH}%.o : %.C ${INCLUDES} | dir date
 # *****************************************************************************
 # Create the Target
 # *****************************************************************************
-LINK_LIBS := -L${PDBG_ROOT}/.libs -lpdbg -lfdt -lz -lyaml
+ifeq (${EDBG_ISTEP_CONTROL}, yes)
```

Comment:
This is not correct - the variable names in the conditionals don't line up.  Just do this:
```
LINK_LIBS := -L${PDBG_ROOT}/.libs -lpdbg -lfdt -lz -lyaml
ifeq (${EDBG_ISTEP_CONTROL}, yes)
    LINK_LIBS += -lipl
```